pkgs/applications/office/elementary-planner/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkgconfig, appstream-glib, desktop-file-utils
+, python3, vala, wrapGAppsHook
+, evolution-data-server
+, libical
+, libgee
+, json-glib
+, sqlite
+, libsoup
+, gtk3
+, pantheon /* granite, schemas */
+, webkitgtk
+, appstream
+}:
+
+stdenv.mkDerivation rec {
+  pname = "elementary-planner";
+  version = "2.1.1";
+  src = fetchFromGitHub {
+    owner = "alainm23";
+    repo = "planner";
+    rev = version;
+    sha256 = "0swj94pqf00wwzsgjap8z19k33gg1wj2b78ba1aj9h791j8lmaim";
+  };
+
+  nativeBuildInputs = [
+    appstream-glib
+    desktop-file-utils
+    meson
+    ninja
+    pkgconfig
+    python3
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    evolution-data-server
+    libical
+    libgee
+    json-glib
+    sqlite
+    libsoup
+    gtk3
+    pantheon.granite
+    webkitgtk
+    appstream
+    pantheon.elementary-gsettings-schemas
```

Comment:
this is actually only used in `NIX_GSETTINGS_OVERRIDES`. There's no special schemas.
I probably should have named it better

Reply:
```suggestion
```

Reply:
Actually I noticed the result never retained references to this anyway, ha but wasn't sure why not and figured better safe than sorry. Now I know!  All good re:naming, will keep that in mind.